test/test_sparse_csr.py

```diff
@@ -360,6 +365,19 @@ def test_coo_to_csr_convert(self, device, dtype, coalesced):
 
         self.assertEqual(coo.matmul(vec), csr.matmul(vec))
 
+        col_indices = torch.tensor([
+            31, 92, 65, 50, 34, 62, 22, 56, 74, 89
+        ], dtype=torch.int64, device=device)
+        self.assertEqual(csr.col_indices(), col_indices)
+
+        self.assertEqual(csr.crow_indices().shape[0], 101)
+        self.assertEqual(csr.crow_indices().min(), 0)
+        self.assertEqual(csr.crow_indices().max(), 10)
+        self.assertTrue(all(csr.crow_indices()[1:] >= csr.crow_indices()[:-1]))
```

Comment:
This test is a CSR tensor invariance test (see https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/sparse/SparseCsrTensor.cpp#L77-L81) and hence superfluous here.